### DISCUSSION TITLE
Own Servericon in maintenance mode (expansion of maintenance command)

### DISCUSSION THREAD

**Discussion by CloudeLecaw** [Zrips/CMI#6983] [Enhancement]
When you look at the server list the server with its icon looks like this.

![image](https://user-images.githubusercontent.com/12080017/174164099-9c5019f7-6c45-4ed9-a9c0-c1d29d226d2d.png)

It would be cool if this server icon could be changed if the maintenance in enabled

So the servericon could be changed until its enabled like this

![image](https://user-images.githubusercontent.com/12080017/174164457-ad85f601-e19e-489f-bc16-6cfa0539397b.png)

Well you see the icon now changed.
When the maintenance is disabled it should change to its default icon.
![image](https://user-images.githubusercontent.com/12080017/174164099-9c5019f7-6c45-4ed9-a9c0-c1d29d226d2d.png)

With a custom alias its allredy possible to change the shown modt to a text that tells that the server is in  maintenance and change it also back with a custom alias.  the only thing that does not work (or i dont know how...) is changing the shown servericon by command.

So it would be nice if this could be added to switch the shown Servericon. Also a command for that would be a possibility like... **/cmi servericon picturename.png**   The picture may could be place in the main direcoty where the default server-icon.png allready is placed... or in a cmi folder (cmi allredy had this nice images folder...

**Comment by mrfloris** [comment]
This would be pretty cool if possible. `+1`

**Comment by CloudeLecaw** [comment]
Well as longer as i thing about maybe it also could be a command argument of the maintenance command

cmi maintenance (true/false) **(-icon:picturename.png)** (message)
-icon:picturename is optional like -p: -s etc in other commands and would change the shown servericon.

when disabling the maintenance you could set it back by setting the default picturesname in the custom alias.

Note: currently the state of maintanace is saved also when the server restarts. this means also the chosen servericon still should be
shown. So maybe it could be a little new config entry in the servermaintance section of the config.yml...
